src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2602,24 +2602,26 @@
      *
      * <p>For example:</p>
      * <pre>
-     * (quantity == 1) ? "": "s"
+     * String variable=(quantity==1)?"true":"false";
      * </pre>
-     * <p>
-     * parses as:
-     * </p>
+     * <p>parses as:</p>
      * <pre>
-     * +--QUESTION (?)
-     *     |
-     *     +--LPAREN (()
-     *     +--EQUAL (==)
-     *         |
-     *         +--IDENT (quantity)
-     *         +--NUM_INT (1)
-     *     +--RPAREN ())
-     *     +--STRING_LITERAL ("")
-     *     +--COLON (:)
-     *     +--STRING_LITERAL ("s")
-     * </pre>
+     * |--TYPE -&gt; TYPE
+     * |   `--IDENT -&gt; String
+     * |--IDENT -&gt; variable
+     *     `--ASSIGN -&gt; =
+     *     `--EXPR -&gt; EXPR
+     *     `--QUESTION -&gt; ?
+     *  |--LPAREN -&gt; (
+     *  |--EQUAL -&gt; ==
+     *  |   |--IDENT -&gt; quantity
+     *  |   `--NUM_INT -&gt; 1
+     *  |--RPAREN -&gt; )
+     *  |--STRING_LITERAL -&gt; "true"
+     *  |--COLON -&gt; :
+     *      `--STRING_LITERAL -&gt; "false"
+     *  --SEMI -&gt; ;
```

Comment:
Please copy ast structure from CLI output without modifications, please read good practice link and review commits for similar task that are already merged for a lot of examples

Reply:
Thanks for your time @romani for reviewing my PR, I will correct it as required, apologize for inconvenience caused by me.

Reply:
> Please copy ast structure from CLI output without modifications, please read good practice link and review commits for similar task that are already merged for a lot of examples

Please review @romani, I corrected ast structure and also PR according to that.
Thanks in advance..

Reply:
> > Please copy ast structure from CLI output without modifications, please read good practice link and review commits for similar task that are already merged for a lot of examples
> 
> Please review @romani, I corrected ast structure and also PR according to that.
> Thanks in advance..

Please review @romani...
Thanks in advance..